docs/Using-Fleet/REST-API.md

```diff
@@ -5170,6 +5172,14 @@ Returns a list of all queries in the Fleet instance.
     "name": "query1",
     "description": "query",
     "query": "SELECT * FROM osquery_info",
+    "team_id": null,
+    "frequency": 3600,
+    "platform": null,
+    "min_osquery_version": null,
+    "automations": {
+        "logging": "differential",
+        "ignore_removal": true
+    },
```

Comment:
Currently this kind of data (for policies) lives in  the "find one team" endpoint results  
<img width="330" alt="image" src="https://github.com/fleetdm/fleet/assets/618009/634cd693-c015-4e6f-8563-05aaf3a84ef8">


For scheduled queries, it lives alongside as a sibling